### PR TITLE
Fix https://github.com/tools-life/taskwiki/issues/413

### PR DESCRIPTION
**Pull request by newptcai**
Fixes tools-life/taskwiki#413

### CODE REVIEW

File: taskwiki/viewport.py [hunk 1]
```diff
@@ -96,7 +96,7 @@ def process_filterstring(self, filterstring, use_presets=True):
         # Process syntactic sugar: Context expansion
         detected_contexts = []
         for token in filter(lambda x: x.startswith('@'), taskfilter_args):
-            context_variable_name = 'context.{0}'.format(token[1:])
+            context_variable_name = 'context.{0}.read'.format(token[1:])
             context_definition = self.tw.config.get(context_variable_name)
```
Comment (liskin) [inline taskwiki/viewport.py:100]: This will break old-style contexts :-(

Comment (newptcai) [reply taskwiki/viewport.py:100]: I use Taskwarirro 2.6.1 with the following context
```
task context define computer +computer or +todo or +journal
```
This change works well with such definition.
I have not used older versions of context so I am not sure how that works.

Comment (liskin) [reply taskwiki/viewport.py:100]: Yeah, new taskwarrior sets `context.<name>.read` on `task context define`, but old used to set just `context.<name>` and it's still supported in .taskrc, so we should support both here.

Comment (newptcai) [reply taskwiki/viewport.py:100]: OK. How about a try-catch approach? If one format fails we try another?

Comment (liskin) [reply taskwiki/viewport.py:100]: Yes, a fallback mechanism would be best. Ideally with the same semantics as to which definition is preferred as taskwarrior itself.

**Comment by newptcai** [comment]
I will make a new pull request.

**Pull request closed without merging**